edward/util/random_variables.py

```diff
@@ -778,3 +778,54 @@ def transform(x, *args, **kwargs):
   new_x = TransformedDistribution(x, bij, *args, **kwargs)
   new_x.support = new_support
   return new_x
+
+
+def softmax(vec):
```

Comment:
By convention, any functions without a leading underscore in `util/` are exposed to the user. Since you only apply this function once it could make sense to just write it in-line.

Reply:
Makes sense. Moved this into `compute_multinomial_mode`. Left it as a function, as it's used in the `while` loop.